pages/_app.tsx

```diff
@@ -18,27 +19,29 @@ const queryClient = new QueryClient();
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
-    <ErrorProvider>
-      <VoteTimingProvider>
-        <WalletProvider>
-          <QueryClientProvider client={queryClient}>
-            <UserProvider>
-              <ContractsProvider>
-                <BalancesProvider>
-                  <VotesProvider>
-                    <PanelProvider>
-                      <GlobalStyle />
-                      <Component {...pageProps} />
-                    </PanelProvider>
-                  </VotesProvider>
-                </BalancesProvider>
-              </ContractsProvider>
-              <ReactQueryDevtools />
-            </UserProvider>
-          </QueryClientProvider>
-        </WalletProvider>
-      </VoteTimingProvider>
-    </ErrorProvider>
+    <DefaultError.Provider>
+      <PanelError.Provider>
```

Comment:
2 error contexts, one for panel, one for everything else

Reply:
nit: can we update this to use the same import pattern as the other contexts just for consistency?